build.sbt

```diff
@@ -52,3 +52,25 @@ lazy val `zio-telemetry` =
     )
 
 testFrameworks := Seq(new TestFramework("zio.test.sbt.ZTestFramework"))
+
+val circeVersion  = "0.12.3"
+val sttpVersion   = "2.0.0-M9"
+val jaegerVersion = "1.0.0"
+val zipkinVersion = "2.11.0"
+
+lazy val examples =
```

Comment:
Please introduce the root project and aggregate on these two (and all of the others). Note that the root is also skipped in publishing.

Optionally, you can move all projects into the modules directory.